02-linked-lists/02-linked-lists.md

```diff
@@ -344,63 +343,49 @@ When designing a data structure like a Linked List, we typically design the data
 
 The node class encapsulates each element of the LinkedList..  It provides an attribute to store data and a node referencing the next node in the chain.  It provides an interface for our LinkedList to deal with the data and connect nodes.
 
-```ruby
+```python
 # Defines a node in the singly linked list
-class Node
-  attr_reader :data # allow external entities to read value but not write
-  attr_accessor :next # allow external entities to read or write next node
-
-  def initialize(value)
-    @data = value
-    @next = nil
-  end
-end
+class Node:
+    def __init__(self, value):
+        self.data = value
+        self.next = None
 ```
```

Comment:
This is my obligatory asking to make this a functional programming style `List` class where the node _is_ the list.  

(I don't feel super strongly about it but as someone that's done FP I feel obligated to ask for the slightly cleaner version.  :P)